scripts/release.py

```diff
@@ -276,12 +293,57 @@ def update_milestones(self):
                 )
             else:
                 raise
+        else:
+            if not milestone_rc:
+                raise Exception(
+                    'None object Returned (Probably a 404 due to auth issues)',
+                    'cannot create milestone'
+                )
         for milestone in self.gh.milestones():
             if milestone.title == self.tag:
                 milestone.update(state='closed')
                 break
 
 
+def chk_devel_version(repo, branch, expected_release):
+    repo.git.checkout(branch)
+    repo.git.pull(repo.remote, branch)
+    for release_file in RELEASE_FILES:
+        filename = os.path.join(repo.path, release_file)
+        with open(filename, 'r') as f:
+            content = f.read()
+        current_release = yaml.load(content)['rpc_release']
+        version_check = os.getenv('RPC_VERSION_CHECK', True)
+        if (current_release != expected_release and
+                version_check not in ['False', 'no', 'FALSE', 'NO']):
```

Comment:
I'm not a fan of this type of validation, there are plenty of ways this could go wrong, what happens if the user sets the var to `"Falsee"` or `0` etc. `version_check` should be one of two values if set otherwise the script should just fail.